Apps/FCM/ios/FCMSampleApp/MyAppPushNotificationsHandler.swift

```diff
@@ -9,8 +8,8 @@ public class MyAppPushNotificationsHandler : NSObject {
 
   public override init() {}
 
-  @objc(setupCustomerIOClickHandling:)
-  public func setupCustomerIOClickHandling(withNotificationDelegate notificationDelegate: UNUserNotificationCenterDelegate) {
+  @objc(setupCustomerIOClickHandling)
```

Comment:
Changed this file to what RN customers will begin using when the iOS feature launches.